internal/cmd/gencli/templates.go

```diff
@@ -516,6 +521,19 @@ func (c *{{ camelCase .SubActionPrefix }}Command) Run(args []string) int {
 	return base.CommandSuccess
 }
 
+func (c *{{ camelCase .SubActionPrefix }}Command) checkFuncError(err error) int {
+	if err == nil {
+		return 0
+	}
+	if apiErr := api.AsServerError(err); apiErr != nil {
+		var opts []base.Option
+		c.PrintApiError(apiErr, fmt.Sprintf("Error from controller when performing %s on %s", c.Func, c.plural), opts...)
```

Comment:
Why declare and then pass in empty opts? Can just leave it empty.

Reply:
just repurposed the code into a helper function so I don't know why. I will remove it.